### PR TITLE
check on yarn production incompatibility with npm

### PR DESCRIPTION
**Pull request by hunterloftis**

### CODE REVIEW

File: lib/dependencies.sh [hunk 1]
```diff
@@ -33,6 +33,15 @@ yarn_node_modules() {
   # according to docs: "Verifies that versions of the package dependencies in the current project’s package.json matches that of yarn’s lock file."
   # however, appears to also check for the presence of deps in node_modules
   # yarn check 1>/dev/null
+  if [ "$NODE_ENV" == "production" ]; then
+    if [ "$NPM_CONFIG_PRODUCTION" == "false" ]; then
```
Comment (hone) [inline lib/dependencies.sh]: can't you just do:

```sh
if [ "$NODE_ENV" == "production" ] && [ "$NPM_CONFIG_PRODUCTION" == "false" ]; then
```

Comment (hunterloftis) [inline lib/dependencies.sh]: Yeah, but I try to keep if statements under 80 chars if I can.

Comment (hunterloftis) [inline lib/dependencies.sh]: But if it'll get your approval faster... ;)

**Comment by hone** [comment]
:+1:

**Comment by hone** [comment]
:+1:

**Pull request merged**